internal/controller/etcd/reconcile_spec.go

```diff
@@ -172,17 +200,40 @@ func (r *Reconciler) getOrderedOperatorsForPreSync() []component.Kind {
 	return []component.Kind{}
 }
 
-func (r *Reconciler) getOrderedOperatorsForSync() []component.Kind {
-	return []component.Kind{
-		component.MemberLeaseKind,
-		component.SnapshotLeaseKind,
+func (r *Reconciler) getOrderedOperatorsForSync(etcdObjMeta metav1.ObjectMeta) []component.Kind {
+	var operators []component.Kind
+
+	if druidv1alpha1.IsEtcdRuntimeComponentCreationEnabled(etcdObjMeta) {
+		operators = []component.Kind{
+			component.ServiceAccountKind,
+			component.RoleKind,
+			component.RoleBindingKind,
+			component.MemberLeaseKind,
+			component.SnapshotLeaseKind,
+		}
+	}
+
+	// add the rest of the operators that are always needed for the etcd cluster
+	operators = append(operators,
```

Comment:
is client and peer service really required in case of autonomous cluster use case? Similarly is PDB component required as these are static pods where does PDB fit?

Reply:
Valid points. PDB is definitely not required. The service could still be used by the translator to expose specific ports from the etcd static pods. 
@rfranzke are client and peer service resources created by druid used in the ASC setup?

Reply:
Sure, thanks Rafael.